src/rust/ensogl/lib/components/src/list_view.rs

```diff
@@ -0,0 +1,14 @@
+//! ListView EnsoGL Component.
+//!
+//! ListView a displayed list of entries with possibility of selecting one and "chosing" by
+//! clicking or pressing enter - similar to the HTML `<select>`.
+
+pub mod component;
+pub mod entry;
+
+/// Commonly used types and functions.
+pub mod prelude {
+    pub use ensogl_core::prelude::*;
+}
```

Comment:
1. The prelude module should be only on top/crate level.
2. I would move the stuff from `component` module to this file (before I was reluctant to keep much code in lib.rs, but this is not lib.rs anymore).